Rakefile

```diff
@@ -0,0 +1,76 @@
+#
+# Cookbook Name:: python-webapp
+#
+# Copyright Chef, OSU Open Source Lab
```

Comment:
This should be changed from ``OSU Open Source Lab`` to ``Oregon State University``. Please change this in all places referenced in the copyright header.